### ISSUE TITLE
[ADD EXAMPLE] Julia set

### ISSUE THREAD

**Issue by jeremydouglass** [jeremydouglass/rosetta_examples_p5#24]
### URL of task
https://rosettacode.org/wiki/Julia_set#Processing

### Next steps
Create and submit a fully commented Processing sketch based on the linked example -- either as an issue comment or as a pull request. See CONTRIBUTING.md for details.

**Comment by villares** [comment]
What do you think about removing the hsv2rgb() function from the example at the wiki?
Is it ok to discuss this here? I feel it there is more people here than on the discuss page...

```java
void setup() {
  size(640, 480);
}

float cX = -0.7;
float cY = 0.27015;
float zx, zy;
float maxIter = 300;

void draw() {
  for (int x = 0; x < width; x++) {
    for (int y = 0; y < height; y++) {
      zx = 1.5 * (x - width / 2) / (0.5 * width);
      zy = (y - height / 2) / (0.5 * height);
      float i = maxIter;
      while (zx * zx + zy * zy < 4 && i > 0) {
        float tmp = zx * zx - zy * zy + cX;
        zy = 2.0 * zx * zy + cY;
        zx = tmp;
        i -= 1;
      }
      colorMode(HSB);
      color c = color(i / maxIter * 255, 255,  i > 1 ? 255 : 0);
      //color c = hsv2rgb(i / maxIter * 360, 1, i > 1 ? 1 : 0);
      set(x, y, c);
    }
  }
  noLoop();
}
```

**Comment by villares** [comment]
This is would be my Python mode port.
Note that for a single frame thing I would draw from setup perhaps... 

```python
cX = -0.7
cY = 0.27015
maxIter = 300

def setup():
    size(640, 480)
    for x in range(width):
        for y in range(height):
            zx = 1.5 * (x - width / 2) / (0.5 * width)
            zy = (y - height / 2) / (0.5 * height)
            i = maxIter
            while zx * zx + zy * zy < 4 and i > 0:
                tmp = zx * zx - zy * zy + cX
                zy = 2.0 * zx * zy + cY
                zx = tmp
                i -= 1
            colorMode(HSB)
            c = color(i / maxIter * 255, 255, 255 if i > 1 else 0)
            set(x, y, c)
```

**Comment by jeremydouglass** [comment]
I agree that `colorMode(HSB)` is the native Processing way of drawing with hue. The changes on the wiki look good.

Keep in mind that you can ping people on the wiki -- on the diiscussion page, or on their talk pages -- using @. So if somebody made a change there and you want to ask them about it, that is possible.

That said, I agree that the wiki doesn't seem to have a very active discussion community going on, and I have always found wiki talk pages extremely frustrating. If you have come up with something that you think is a clear improvement, just do it -- and say why in the change comment ("do hue using native colorMode"). It can always be reverted from history if someone disagrees.

**Comment by jeremydouglass** [comment]
Closed by #60 (both Java and Python)
